src/web/pages/users/dialog.js

```diff
@@ -48,45 +52,73 @@ import Divider from 'web/components/layout/divider';
 import Layout from 'web/components/layout/layout';
 
 class Dialog extends React.Component {
+  dataToSave = false;
```

Comment:
Is this only the initialization of the variable? Please make sure, the type matches whatever the variable will receive later on. I'd suspect that `{}` would better fit. However, this could lead to the situation where you send an empty object, instead of the actual user data. Same is true for the `false` by the way.
I'm kind of missing where the real data is send here.